### DISCUSSION TITLE
关于baseapp一直报错 TCPPacketSender::processSend

### DISCUSSION THREAD

**Discussion by qq952670771** [kbengine/kbengine#1344] [Q&A]
如题：云服务器上的baseapp一直报错
119.98.188.14:58299: TCPPacketSender::processSend(internal, 1): Exception occurred: REASON_RESOURCE_UNAVAILABLE
请问一下这个是什么原因啊？

**Comment by kbengine** [comment]
包无法发送出去， tcp缓冲区填满了

**Reply by qq952670771** [reply]
那请问一下，如何处理这种情况啊？目前也就一个玩家在战斗，都有这个报错

**Reply by kbengine** [reply]
要么发包太快， 要么客户端处理太慢， 或者网络问题  你自己分析

**Reply by qq952670771** [reply]
可能是发包太快的原因，因为做性能测试的时候，10s，至少发了700次包，对于这种情况请问一下，怎么办啊？
